docs/libraries.md

```diff
@@ -20,7 +20,7 @@ repositories {
 }
 ```
 
-You now add a maven repository with the URL you found to it:
+You now add a maven repository with the URL that links to it:
```

Comment:
```suggestion
You now need to add the maven repository's URL to the section:
```

Reply:
Yeah I was most unsure on how to word that because it made little sense to me in the first place :P